docs/reference/runtime-fields.asciidoc

```diff
@@ -1,15 +1,17 @@
-[[alerts-runtime-fields]]
-== Add runtime fields to detection alerts
+[[runtime-fields]]
+== Create runtime fields in {elastic-sec}
 
 Runtime fields are fields that you can add to documents after you've ingested your data. For example, you could combine two fields and treat them as one, or perform calculations on existing data and use the result as a separate field. Runtime fields are evaluated when a query is run. 
 
-You can add a runtime field to your detection alerts from the *Alerts* table. The new field applies to all {elastic-sec} alerts.
+You can create a runtime field and add it to your detection alerts or events from any page that lists alerts or events in a data grid table, such as *Alerts*, *Timelines*, *Hosts*, and *Users*. Once created, the new field becomes available to all {elastic-sec} alerts and events.
 
 NOTE: Runtime fields can impact performance, because they're evaluated each time a query runs. Refer to {ref}/runtime.html[Runtime fields] for more information.
```

Comment:
Not your fault, just a slight nit here. 
```suggestion
NOTE: Runtime fields can impact performance because they're evaluated each time a query runs. Refer to {ref}/runtime.html[Runtime fields] for more information.
```